lib/addons/istio-addons/istio-cni.ts

```diff
@@ -0,0 +1,27 @@
+import { Construct } from 'constructs';
+import { ClusterInfo } from "../../spi";
+import { HelmAddOn, HelmAddOnProps } from "../helm-addon";
+import { dependable, supportsALL } from '../../utils';
+
+const defaultProps: HelmAddOnProps = {
+    name: 'istio-cni',
+    release: 'cni',
+    namespace: 'istio-system',
+    chart: 'cni',
+    version: "1.20.1",
+    repository: 'https://istio-release.storage.googleapis.com/charts',
+    values: {}, 
+};
+
+@supportsALL
+export class IstioCniAddon extends HelmAddOn {
+
+    constructor() {
```

Comment:
why is it empty? How will customers accomplish the following use cases:

1. Pass values to this helm chart from the blueprint. 
2. Discover what values are possible to pass. Generally we put most popular choices to the props object.
3. Override the version of helm chart to be anything other than the hard-coded 1.20.1. e.g. I am sure once 1.20.2 is out they will want to use that. 

Applies to all new addons added in the PR.

Reply:
Fixed